inference-engine/tests/functional/shared_test_classes/src/single_layer/deformable_convolution.cpp

```diff
@@ -48,7 +48,8 @@ InferenceEngine::Blob::Ptr DeformableConvolutionLayerTest::GenerateInput(const I
         if (name == "a_data") {
             blobPtr = LayerTestsUtils::LayerTestsCommon::GenerateInput(info);
         } else if (name == "b_offset_vals") {
-            blobPtr = FuncTestUtils::createAndFillBlob(info.getTensorDesc(), 2, 0, 0.1);
+            // TODO: createAndFillBlob() resolution is int, need to rewrite it to test also real number offsets
```

Comment:
// TODO: createAndFillBlob() resolution is int, need to rewrite it to test also real number offsets

Reply:
You can try `createAndFillBlobFloat` or `createAndFillBlobWithFloatArray`

Reply:
Done